plugins/gtav/Game.cpp

```diff
@@ -5,113 +5,79 @@
 
 #include "Game.h"
 
-#include "mumble_positional_audio_utils.h"
+#include <sstream>
 
-#include <cstring>
+Game::Game(const uint64_t id, const std::string &name) : m_proc(static_cast< procid_t >(id), name) {
+}
+
+bool Game::setupPointers(const Module &module) {
+	// 48 8B 0D ?? ?? ?? ??    mov    rcx, cs:off_????????
+	const std::vector< uint8_t > playerMgrPattern = { 0x48, 0x8B, 0x0D, '?', '?',  '?',  '?',  0x8A, 0xD3, 0x48, 0x8B,
+													  0x01, 0xFF, 0x50, '?', 0x4C, 0x8B, 0x07, 0x48, 0x8B, 0xCF };
+
+	m_playerMgr = m_proc.findPattern(playerMgrPattern, module);
+	if (!m_playerMgr) {
+		return false;
+	}
+
+	// 48 8B 05 ?? ?? ?? ??    mov    rax, cs:qword_????????
+	const std::vector< uint8_t > cameraMgrPattern = { 0x48, 0x8B, 0x05, '?', '?', '?', '?', 0x4A, 0x8B, 0x1C, 0xF0 };
+	m_cameraMgr                                   = m_proc.findPattern(cameraMgrPattern, module);
+	if (!m_cameraMgr) {
+		return false;
+	}
+
+	m_playerMgr = m_proc.peekPtr(m_proc.peekRIP(m_playerMgr + 3));
+	m_cameraMgr = m_proc.peekRIP(m_cameraMgr + 3);
```

Comment:
Another magic number (`3`) that I think should get a name

Reply:
It's in the signature right above, see the question marks.